### ISSUE TITLE
Make doors open more easily for people who have trouble with doors

### ISSUE THREAD

**Issue by ajlennon** [DoESLiverpool/somebody-should#645]
Please make your issue:
* Specific

As above

* Measurable

As above

* Achievable

I think it's pretty standard stuff if we don't let scope creep stop us

* Responsible

Me (if DoES community see this of value) + anybody else who will help.

* Time-constrained

Within a couple of weeks of moving into the new Tapestry building.

---

I was speaking with @robotorium tonight about bits and pieces and it came up that it might be pretty nice to have doors that opened automatically for people who have trouble with doors. God knows I had to get myself chipped in the hand to be able to get in with less fuss so I can imagine if you had more trouble than me it could be frustrating.

So this is a call for discussion as to whether this idea works.

(a) In the simplest case I'm thinking automatically opening doors internally as I think that's pretty well understood stuff?

(b) In the more complex techie case RFID tags and such that automatically open to give access 

But if we look at the way people use the space internally to move from room to room, workshop, kitchen, toilet etc.?

Also @DoESsean perhaps there is funding for this?

**Comment by Robotorium** [comment]
Doors are always a struggle, and while everyone at DoES is always super helpful, it's frustrating to have to ask for help and/or figure out some way to struggle through. If it's possible, it would be really cool to have this (but obviously it's doable without if necessary).
Be a good excuse to get chipped though...

**Comment by johnmckerrell** [comment]
This is certainly something we're aware of, it was always something we missed that the lifts in Gostins might be great but you've got to get through two difficult doors to get to them. I'll need to contact the landlord to see if it's been remembered though.

**Comment by ajlennon** [comment]
Can I throw this over to you for the moment thanks @johnmckerrell ?

**Comment by ajlennon** [comment]
@Robotorium we'll chat about getting you chipped after the doors are sorted :) I don't want to be the only one!

**Comment by amcewen** [comment]
We've been talking to the door-security-access people about "proper" automatic door openers, but as a [couple](https://blog.arduino.cc/2018/01/12/automatically-open-doors-with-arduino/) of [DIY options](https://blog.adafruit.com/2018/01/12/low-cost-iot-door-opener-for-disabled-user/) have crossed my RSS reading of late, I figured it wouldn't hurt to note them here.

**Comment by johnmckerrell** [comment]
We got a price back from the "door" people of £3700 per door. Right now that doesn't feel like something we could cover but it's good to have an idea of what a paid-for solution would cost. Possibly we can look at one of the solutions Adrian listed.

**Comment by Robotorium** [comment]
If doors that open themselves aren't doable, could it be possible to have
doors that open both inward and outward? Gives more options for how to get
through.

Op di 17 apr. 2018 21:35 schreef John McKerrell <notifications@github.com>:

**Comment by johnmckerrell** [comment]
Hm.. interesting. Right now that wouldn't work on the doors that have mag locks (as the mag lock would physically stop that). I have a feeling that the doors are not particularly heavy to move although I keep forgetting to check in-situ.

**Comment by johnmckerrell** [comment]
For the record the front doors actually are quite heavy, I don't think the internal doors are though. I have been wondering about the DIY options mentioned above. I'm sure @goatchurchprime would know a thing or two about heavy duty stepper motors.

**Comment by Robotorium** [comment]
Yeah the front doors are hard, I haven't had to try it without help yet but
it would be a struggle

Op di 14 aug. 2018 13:44 schreef John McKerrell <notifications@github.com>:

**Comment by DoESsean** [comment]
Has there been any movement on the idea of some DIY solution to this? @Robotorium have you found any issues that we can look to try and solve?

**Comment by amcewen** [comment]
It seems that our new neighbours in the escape room downstairs are planning on fitting electric door openers to the front doors.  More news when we know it.
